mega-zep-frontend/src/main/angular/frontend/src/app/modules/monthly-report/components/general-info/general-info.component.ts

```diff
@@ -0,0 +1,59 @@
+import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+import {MonthlyReport} from '../../models/MonthlyReport';
+
+const identifiers: string[] = [
+  'Arbeitszeit gesamt',
+  'fakturierbare Stunden',
+  'Verrechenbarkeit',
+  'Urlaub',
+  'Zeitausgleich',
+  'Homeoffice'
+];
```

Comment:
Move definition into class and access values directly over this field instead of assigning it to a new variable in constructor